CONTRIBUTING.md

```diff
@@ -35,3 +35,9 @@ Then, update `_data/content.yml` with the name of your new lesson under the appr
 If you've added a new section (i.e. a new directory under `/lessons`), add the section name under the `sections` key of `_data/locales/en.yml` or `_data/locales/<language_code>.yml` if the section + lesson you added are not in English.
 
 Thank you for your contributions!
+
+## Gotcha
```

Comment:
Should add something here about running Jekyll locally before pushing?  

```shell
➜  elixirschool git:(master) bundle exec jekyll s
Configuration file: /Users/scallan/Projects/elixirschool/_config.yml
            Source: /Users/scallan/Projects/elixirschool
       Destination: /Users/scallan/Projects/elixirschool/_site
 Incremental build: disabled. Enable with --incremental
      Generating...
  Liquid Exception: Liquid syntax error (line 183): Tag '{%' was not properly terminated with regexp: /\%\}/ in /Users/scallan/Projects/elixirschool/_posts/2018-11-29-jwt-auth-with-joken.md
jekyll 3.8.5 | Error:  Liquid syntax error (line 183): Tag '{%' was not properly terminated with regexp: /\%\}/
```

Reply:
ah yes let's!

Reply:
@doomspork done

Reply:
leaving this section out b/c now that we have Travis set up to run `jekyll build` this isn't necessary to avoid breaking the site.